### DISCUSSION TITLE
How can I export my old settings to a new installalation of opentrack?

### DISCUSSION THREAD

**Discussion by Cliff-1** [opentrack/opentrack#1575] [Q&A] [answered]
How can I export my old config file with all my settings and preferences on it to a new installation of opentrack.

**Comment by sthalik** [comment, marked as answer]
Settings are stored in `Documents/opentrack-2.3`. You can also open the profile directory from the dropdown to the left of the profile combobox.

With the exception of game detector settings, which are stored in the registry.
